doc/source/handycontrol/native_controls/comboBox/index.md

```diff
@@ -29,7 +29,13 @@ title: ComboBox 组合框
 {% note warning %}
 标题在左时，为了多个输入框左侧对齐，需要设置标题宽度，标题宽度无需逐个设置，可在外部容器上统一设置。
 {% endnote %}
-
+{% note waining %}
```

Comment:
@guanguanchuangyu  waining must be warning

Reply:
yes, you are right 😆